README.md

```diff
@@ -284,6 +284,47 @@ Content-ID: <body@here>
 </html>
 ```
 
+# Attachments
+This example shows how to add attachments with a special `ATTACHMENT` header.
+
+**mailmerge_template.txt**
+```
+TO: {{email}}
+SUBJECT: Testing mailmerge
+FROM: My Self <myself@mydomain.com>
+ATTACHMENT: file1.docx
+ATTACHMENT: ../file2.pdf
+ATTACHMENT: /z/shared/{{name}}_submission.txt
+
+Hi, {{name}},
+
+This email contains three attachments.
+Pro-tip: Use Jinja to customize the attachments based on your database!
+```
+
+Dry run to verify attachment files exist. If an attachment filename includes a template, it's a good idea to dry run with the `--no-limit` flag.
+```shellsession
+$ mailmerge
+>>> message 0
+TO: myself@mydomain.com
+SUBJECT: Testing mailmerge
+FROM: My Self <myself@mydomain.com>
+
+Hi, Myself,
+
+This email contains three attachments.
+Pro-tip: Use Jinja to customize the attachments based on your database!
+
+>>> encoding ascii
+>>> attached /Users/awdeorio/Documents/test/file1.docx
+>>> attached /Users/awdeorio/Documents/file2.pdf
+>>> attached /z/shared/Myself_submission.txt
+>>> sent message 0 DRY RUN
+>>> This was a dry run.  To send messages, use the --no-dry-run option.
+```
+
+Note: The Unix-style home directory specifier (`~`) is not supported.
```

Comment:
Could this be solved with https://docs.python.org/3.6/library/os.path.html#os.path.expanduser ?

Reply:
Haha, I should realize by now that there's an easy way to solve a lot of things in Python. Updated in `bbf8e2d`.